### PR TITLE
 Removing validate node inventory step from pre process.

### PR DESCRIPTION
**Pull request by panchn**

### CODE REVIEW

File: dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/service/delegates/InventoryNode.java [hunk 1]
```diff
@@ -76,11 +79,16 @@ public void delegateExecute(final DelegateExecution delegateExecution)
             final String message = "Update Node Inventory request failed.";
             updateDelegateStatus(message, ex);
             nodeDetails.stream().filter(Objects::nonNull).forEach(nodeDetail -> nodeDetail.setInventoryFailed(true));
```
Comment (aroberson) [inline dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/service/delegates/InventoryNode.java]: this is no longer needed,

File: dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/service/delegates/InventoryNode.java [hunk 2]
```diff
@@ -102,19 +111,34 @@ public void delegateExecute(final DelegateExecution delegateExecution)
                 }
                 if (!isNodeInventorySaved)
                 {
-                    final String message = "Update Node Inventory on Node with uuid " + entry.getKey() + " failed.";
-                    updateDelegateStatus(message);
-                    nodeDetails.stream().filter(Objects::nonNull).forEach(nodeDetail -> nodeDetail.setInventoryFailed(true));
+                    failedToSaveNodes.add(entry.getKey());
+                    nodeDetails.stream().filter(Objects::nonNull).filter(nodeDetail -> entry.getKey().equals(nodeDetail.getId())).findAny()
```
Comment (aroberson) [inline dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/service/delegates/InventoryNode.java]: Please remove the inventory failed flag.

File: dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/service/delegates/InventoryNode.java (hunk 1 above)
Comment (panchn) [reply dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/service/delegates/InventoryNode.java]: Done.

File: dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/service/delegates/InventoryNode.java (hunk 2 above)
Comment (panchn) [reply dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/service/delegates/InventoryNode.java]: Done.

**Pull request merged**